doc/source/whatsnew/v0.9.1.rst

```diff
@@ -59,7 +59,7 @@ New features
 
         .. ipython:: python
 
-    	    df = DataFrame(np.random.randn(5, 3), columns = ['A','B','C'])
+    	    df = pd.DataFrame(np.random.randn(5, 3), columns = ['A','B','C'])
```

Comment:
Please revert this change (the CI is having issue unrelated to this PR)

Reply:
It was raising a warning on my machine when building but does not seem to be the case on the builder in GH so reverted